src/components/elements/inputs/dropdown.js

```diff
@@ -2,14 +2,10 @@ import React from 'react'
 import Select from 'react-select'
 import Input from './input'
 
-export default class DropdownList extends Input {
+export default class Dropdown extends Input {
```

Comment:
http://demos.telerik.com/aspnet-mvc/dropdownlist/index
https://msdn.microsoft.com/en-us/library/system.web.ui.webcontrols.dropdownlist%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396

I think **DropdownList** is a pretty established name; it is a dropdown with a list of items that you can select. Simply **Dropdown** to be, could imply a dropdown of anything; dropdown of an image, paragraph, form (maybe it's more common to call these Popups though).